src/core_plugins/vega/public/vega_editor_template.html

```diff
@@ -0,0 +1,75 @@
+<div class="vegaEditor" ng-controller="VegaEditorController">
+  <div
+    ui-ace="{
```

Comment:
We need `kbn-ui-ace-keyboard-mode` attached to this, and `import 'ui/accessibility/kbn_ui_ace_keyboard_mode';` in the `vega_editor_controller.js` file.